docs/index.rst

```diff
@@ -0,0 +1,23 @@
+.. PyLmod documentation master file, based on the config in
+   Orcoursetrion created by
+   sphinx-quickstart on Sun Feb 15 13:38:17 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. include:: ../README.rst
+
+Table of Contents
+=================
+.. toctree::
+   :maxdepth: 3
```

Comment:
Weird, I was expecting to have the actual class names show up here.  Does bumping it one do that, or am I misunderstanding the attribute?